scripts/csv-usage-data/usage_csv.py

```diff
@@ -0,0 +1,134 @@
+from datetime import datetime, timedelta, timezone
+from pprint import pprint
+import requests, argparse, os
+import random
+import uuid
+import os
+import csv
+import json
+import re 
+import random
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--days_of_data', type=int, required=False, default=33)
+parser.add_argument('--event_type', type=str, required=False, default="event_type")
+parser.add_argument('--filename', type=str, required=True, default="example_usage.csv")
+
+token = os.getenv('API_TOKEN')
+if not token:
+    raise ValueError("Set API_TOKEN environment variable to metronome API key")
+
+args = parser.parse_args()
+
+days_of_data = args.days_of_data
+if days_of_data > 33 or days_of_data < 1:
+    raise ValueError("days_of_data needs to be greater than 1 and less than 33")
+
+filename = args.filename
+if not isinstance(filename, str):
+    raise ValueError("filename must be a string")
+
+event_type = args.event_type
+if not isinstance(event_type, str):
+    raise ValueError("event_type must be a string")
+
+
+csvFilePath = "./"+filename
+hours_of_data = days_of_data*24
+total_events= []
+
+with open(csvFilePath) as csv_file:
+    csv_reader = csv.reader(csv_file, delimiter = ',')
+    list_of_column_names = []
+    for row in csv_reader:
+        list_of_column_names.append(row)
+        break
+
+def make_json(csvFilePath):
+    data = []
+     
+    with open(csvFilePath, encoding='utf-8') as csvf:
+        csvReader = csv.DictReader(csvf)
+         
+        for rows in csvReader:
+             
+            data.append(rows)
+            rows
+        for items in data:
+            time_delta = random.randint(0,hours_of_data)
+            t = datetime.now(timezone.utc) - timedelta(hours=time_delta)
+            properties = {}
+            itemforproperties = {}
+            for propertiesItems in list_of_column_names[0]:
+                if str(propertiesItems) == "customer_id":
+                    customer_id = items[propertiesItems]
```

Comment:
not sure why this is needed.  you dont use the `customer_id` var anywhere

Reply:
oh, this is because a user can either have the customer_id in the script or they can enter it as a command line argument